src/SkyApm.Diagnostics.SqlClient/SqlClientDiagnosticProcessor.cs

```diff
@@ -75,5 +77,43 @@ public void ErrorExecuteCommand([Property(Name = "Exception")] Exception ex)
                 _tracingContext.Release(context);
             }   
         }
+        #endregion
+
+        #region Microsoft.Data.SqlClient
+        //https://docs.microsoft.com/en-us/dotnet/api/microsoft.data.sqlclient?view=sqlclient-dotnet-core-2.1
+
+        [DiagnosticName(SqlClientDiagnosticStrings.SqlBeforeExecuteCommand)]
```

Comment:
Here should be the `DotNetCoreSqlBeforeExecuteCommand` I think?

Reply:
yes. it has been updated in 2nd commit.  

https://github.com/SkyAPM/SkyAPM-dotnet/pull/387/commits/ef5048b9cb8073dbebefe3ed01d4dd750133f048

The file you checked is marked as Outdated.